common/lib/xmodule/xmodule/video_module/video_handlers.py

```diff
@@ -166,6 +167,10 @@ def get_static_transcript(self, request, transcripts):
         if not self.transcript_language == 'en':
             return response
 
+        # If this video lives in library, the code below is not relevant and will error.
```

Comment:
I'm in no way sure about this without a deeper dive into the import code, but if content libraries can be imported by the command line the same way courses are (which I don't think works), then they can have `static_asset_path` set if the `--nostatic` flag is set.